<Issue id="36175" key="FABN-1075" number="1075" project="10604" reporter="latitiah" assignee="harrisob@us.ibm.com" creator="latitiah" type="10004" summary="Compatibility Test Failure: Status 500 errors with in-file wallet" priority="3" resolution="10001" status="6" created="2018-12-14 17:29:14.0" updated="2019-05-23 16:11:46.0" resolutiondate="2019-05-23 16:11:46.0" votes="0" watches="2" workflowId="47692"> <description><! CDATA Given the following scenarios, it was noted that there were errors when attempting to upgrade a network. - Nodejs Integration Test Suite at 1.3 , Node.js SDK at 1.3, Fabric is at 1.3 & Fabric-CA is at 1.4 - Nodejs Integration Test Suite at 1.3 , Node.js SDK at 1.3, Fabric is at 1.4 & Fabric-CA is at 1.3 - Nodejs Integration Test Suite at 1.3 , Node.js SDK at 1.3, Fabric is at 1.4 & Fabric-CA is at 1.4  Please see FAB-12721 for more details of exact steps to follow and the resulting errors. {code} ***** Network End-to-end flow: invoke transaction to move money using in file system wallet ***** ... not ok 182 Error: No valid responses from any peers. Response from attempted peer comms was an error: {"status":500,"payload":{"type":"Buffer","data":  },"isProposalResponse":true} Response from attempted peer comms was an error: {"status":500,"payload":{"type":"Buffer","data":  },"isProposalResponse":true} at Contract._validatePeerResponses (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/fabric-network/lib/contract.js:18:512) at Contract.submitTransaction (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/fabric-network/lib/contract.js:27:51) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7) --- operator: fail at: test (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/test/integration/network-e2e/invoke.js:526:5) stack: |- Error: Error: No valid responses from any peers. Response from attempted peer comms was an error: {"status":500,"payload":{"type":"Buffer","data":  },"isProposalResponse":true} Response from attempted peer comms was an error: {"status":500,"payload":{"type":"Buffer","data":  },"isProposalResponse":true} at Contract._validatePeerResponses (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/fabric-network/lib/contract.js:18:512) at Contract.submitTransaction (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/fabric-network/lib/contract.js:27:51) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7) at Test.assert  as _assert  (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/node_modules/tape/lib/test.js:224:54) at Test.bound  as _assert  (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/node_modules/tape/lib/test.js:76:32) at Test.fail (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/node_modules/tape/lib/test.js:317:10) at Test.bound  as fail  (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/node_modules/tape/lib/test.js:76:32) at test (/opt/gopath/src/github.com/hyperledger/fabric-sdk-node/test/integration/network-e2e/invoke.js:526:5) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7) ... {code}  ></description> </Issue>
